packages/react-native/BugsnagReactNative.podspec

```diff
@@ -13,9 +13,9 @@ Pod::Spec.new do |s|
   s.author       = { "Bugsnag" => "platforms@bugsnag.com" }
   s.platform     = :ios, "7.0"
   s.source       = { :git => "https://github.com/bugsnag/bugsnag-js.git", :tag => "v#{s.version}" }
-  s.source_files = "ios/BugsnagReactNative/**/*.{h,m}",
+  s.source_files = "ios/BugsnagReactNative/*.{h,m}",
                    "ios/vendor/bugsnag-cocoa/**/*.{h,mm,m,cpp,c}",
-  s.public_header_files = "ios/vendor/bugsnag-cocoa/{#{bugsnag_cocoa_public_header_files.join(',')}}"
+  s.public_header_files = "ios/BugsnagReactNative/Bugsnag/*.{h}"
   s.header_dir = 'Bugsnag'
```

Comment:
did you try with this line removed? It seems somewhat off the beaten path, so would be good if we can remove it. I can't think what problem it solved, and since you've brought in a `Bugsnag/` directory, it might not be required anymore.